chef/cookbooks/ohai/files/default/plugins/linux/s390x.rb

```diff
@@ -0,0 +1,40 @@
+provides "s390x"
+
+if File.exist? "/proc/sysinfo"
```

Comment:
@aspiers we plan to support any architecture other than armv7, aarch64, ppc64le, ppc64, x86_64, i586 ? none of those have that file.